ios/Capacitor/Capacitor/CAPUNUserNotificationCenterDelegate.swift

```diff
@@ -42,14 +42,24 @@ public class CAPUNUserNotificationCenterDelegate : NSObject, UNUserNotificationC
     let request = notification.request
     var plugin: CAPPlugin
     var action = "localNotificationReceived"
-    print(notification)
+    var presentation_options: UNNotificationPresentationOptions = [];
```

Comment:
Swift convention is to use camel case instad of snake case, so should be `presentationOptions`